Automattic/vip-go-mu-plugins
# Update to Two-Factor 0.5.0

The Updated Two-Factor version fixes the issue where users who log-in using WordPress.com SSO are logged out every day because of missing  `rememberme` value.

Related Pull Requests:

* https://github.com/WordPress/two-factor/pull/308
* https://github.com/WordPress/two-factor/pull/316

How to test:

**Prepare:**
1. Create a personal test WordPress.com account without any A8C Privileges
2. Add that user to a test site as an admin but without any special A8C Privileges using WP CLI.
3. Make sure you are not using a special IP
4. Force 2FA if not already enabled.

**Test**
1. "Login with WordPress.com" to a site
2. Confirm that the 2FA dialog form has a hidden `rememberme` value and it's set to `0`
2. Enter 2FA Code
3. Login
4. Confirm that auth cookie expiration is set to `Session`

Apply the PR, repeat the steps above, with the following changes:

- `2.` Confirm that the 2FA dialog form has a hidden `rememberme` value and it's set to `1`
- `4.` Confirm that the auth cookie expiration is set to roughly 1 year from now.
